test/morgan.js

```diff
@@ -211,6 +211,27 @@ describe('morgan()', function () {
           .expect(200, cb)
       })
 
+      it('should handle dates with single digits when using "clf" format', function (done) {
+        var GlobalDate = global.Date
+
+        global.Date = function () { return new GlobalDate('2020-02-27') }
```

Comment:
I'm not proud of this but `sinon` did not play nice with node 0.8 or 0.10 https://travis-ci.org/expressjs/morgan/builds/655979079